### ISSUE TITLE
Add "what is JSCraftCamp" on the homepage

### ISSUE THREAD

**Issue by wolframkriesing** [jscraftcamp/website#1360]
Explain:
- what is a barcamp/openspace
- who can attend
- what can attendees do there?
- how much does it cost?
- link to barcamp
- link to socrates

**Comment by Narigo** [comment]
@wolframkriesing do you have a text for me? :)

**Comment by wolframkriesing** [comment]
```
Welcome to JSCraftCamp - a [BarCamp/OpenSpace/Unconference](https://en.wikipedia.org/wiki/Unconference#Barcamp) event for JavaScript enthusiasts of all levels. BarCamps and OpenSpaces are participant-driven conferences where attendees create the schedule and lead discussions, making it a truly community-driven experience.
JSCraftCamp provides a platform for JavaScript enthusiasts to network and participate in workshops, similar to the [SoCraTes conference](https://www.socrates-conference.de/).
```

**Comment by Narigo** [comment]
Resolved by #1390
